WordPressKit/WordPressRSDParser.swift

```diff
@@ -19,6 +19,10 @@ open class WordPressRSDParser: NSObject, XMLParserDelegate {
         if parser.parse() {
             return endpoint
         }
+        // Return the 'WordPress' API link, if found.
+        if let endpoint {
+            return endpoint
+        }
```

Comment:
It took me a bit to understand the code. I had to expand the source and realize that `parser.parse()` calls a delegate.

I wish there was a clearer API. I suppose we could build one by wrapping `XMLParser` but it might be overkill.